AEFeinstein/Swadge-IDF-5.0
# Port MFPaint

### Description

Adds MFPaint, ported from last year's 1.1 firmware. Also updates the paint UI to make better use of the touchpad and to not be afraid of un-drawing the canvas sometimes.

### Test Instructions

Known issues:
* The save menu is not yet fully moved into the tool wheel but still partially removed so the old one doesn't work either
* The tutorial overlaps the tool wheel and hasn't really been updated yet
* Using the quick menu breaks things. I'll sort it out eventually.
* I bet very bad things will happen if you manage to swap the palette

Tested on the real swadge and the emulator. Crashes seem to be fixed, and the canvas storing seems solid.

So far, only the main draw screen has been updated. Try out the tool wheel! Draw stuff and see if it breaks.

### Ticket Links

Reference #27 


### Readiness Checklist
